types/node/v7/index.d.ts

```diff
@@ -1583,9 +1583,12 @@ declare module "readline" {
         completer?: Completer | AsyncCompleter;
         terminal?: boolean;
         historySize?: number;
+        prompt?: string;
+        crlfDelay?: number;
+        removeHistoryDuplicates?: boolean;
     }
 
-    export function createInterface(input: NodeJS.ReadableStream, output?: NodeJS.WritableStream, completer?: Completer | AsyncCompleter, terminal?: boolean): ReadLine;
+    export function createInterface(input: NodeJS.ReadableStream, output?: NodeJS.WritableStream, completer?: Completer | AsyncCompleter, terminal?: boolean, historySize?: number, prompt?: string, crlfDelay?: number, removeHistoryDuplicates?: boolean): ReadLine;
```

Comment:
Only the first four parameters are actually used, as far as I can tell. 

Before I can accept this PR you would need to commit the change mentioned above and push your changes. Once that is done I can merge.